eng/pipelines/templates/jobs/sdk-build.yml

```diff
@@ -131,30 +133,20 @@ jobs:
           OPENSSL_ENABLE_SHA1_SIGNATURES: 1
 
     ############### TESTING ###############
-    - ${{ if eq(parameters.pool.os, 'windows') }}:
-      - ${{ if eq(parameters.targetArchitecture, 'x64') }}:
-        - powershell: eng/common/build.ps1
-            -restore -test -ci -prepareMachine -nativeToolsOnMachine
-            -configuration $(buildConfiguration)
-            /p:Projects="$(Build.SourcesDirectory)/test/dotnet-new.IntegrationTests/dotnet-new.IntegrationTests.csproj"
-            /p:TestRunnerAdditionalArguments="-class Microsoft.DotNet.Cli.New.IntegrationTests.DotnetNewTestTemplatesTests"
-            ${{ parameters.runtimeSourceProperties }}
-            /bl:$(Build.SourcesDirectory)/artifacts/log/$(buildConfiguration)/TestTemplateTests.binlog
-          displayName: 🟣 Run Test Templates Tests
-          condition: succeeded()
-
     - ${{ if eq(parameters.runTests, true) }}:
       - ${{ if eq(parameters.runAoTTests, true) }}:
         # For the reason this is here, see: https://github.com/dotnet/sdk/issues/22655
         - script: $(Build.SourcesDirectory)/artifacts/bin/redist/$(buildConfiguration)/dotnet/dotnet workload install wasm-tools --skip-manifest-update
           workingDirectory: $(Build.SourcesDirectory)/artifacts/bin
           displayName: 🟣 Install wasm-tools Workload
+
       # For the /p:Projects syntax for PowerShell, see: https://github.com/dotnet/msbuild/issues/471#issuecomment-1146466335
       - ${{ if eq(parameters.pool.os, 'windows') }}:
         - powershell: eng/common/build.ps1
             -restore -test -ci -prepareMachine -nativeToolsOnMachine
             -configuration $(buildConfiguration)
             /p:Projects=\`"${{ replace(parameters.testProjects, ';', '`;') }}\`"
+            /p:TestRunnerAdditionalArguments="${{ parameters.testRunnerAdditionalArguments }}"
```

Comment:
Needed to add a way to pass this value. Since the arguments do not include `;` and `,`, but do contain spaces, the double doubles around the value should be sufficient.